source/tools/hubspot.rst

```diff
@@ -0,0 +1,111 @@
+Hubspot
+============
+
+What is Hubspot?
+----------------
+HubSpot is an "inbound marketing software platform" which offers features for internet marketing, such as emails, blogging and social media. This piece of software is mostly used by the marketing team.
+
+This document will mostly cover tips on how to use the emailer tool/rehabstudio email template.
+
+The rehabstudio email template
+------------------------------
+
+The 2 in 1 template allows you to switch between the event or newsletter email layouts. Each layout includes a spotlight, monthly links and footer section. The spotlight and monthly links can be hidden if not needed.
+
+Name of template:  rehabstudio_new_mailer.html
+
+Getting started on hubspot
+--------------------------
+
+Create a new email using the hubspot template
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To create a new email with Hubspot.
+
+- Click create new email.
+- Select the new template titled rehabstudio_new_mailer.html.
+- Give the email a title, then click create.
+
+The Editor
+------------------
+
+This section allows you to edit the contents of the email template. Each of the contents are stored in modules. You can manipulate the modules by showing or hiding them.
+
+The rehabstudio_new_mailer template has a number of custom modules, with options to either show or hide sections. This will depend on the type of email your building. The options can be found in the ‘Edit Modules’ panel and are highlighted in orange.
+
+Important! - you may have to refresh the page to see changes within the edit module panel and template.
+
+Choose type of email
+~~~~~~~~~~~~~~~~~~~~
+This option will determine if your making either an Event or an Newsletter. Selecting one will change the layout of the email, opening more options for further editing.
+
+- Event
```

Comment:
@KirMcC Keep all of this as a proper bulleted list, rather than using indentation (which makes it look more like a blockquote, as seen here: https://github.com/rehabstudio/devdocs/blob/ced76de9046676c955ba4e49c6081f156b37c63a/source/tools/hubspot.rst#choose-type-of-email)